senders/formatter.go

```diff
@@ -308,3 +314,52 @@ func isUUIDFormat(str string) bool {
 	}
 	return true
 }
+
+func sanitizeInternal(str string) string {
```

Comment:
Could you add some comments to this method and `sanitizeValue` describing what it does? It'd be useful to someone looking at this independently to understand what we're doing.